core/src/main/scala/org/apache/spark/scheduler/local/LocalBackend.scala

```diff
@@ -20,6 +20,7 @@ package org.apache.spark.scheduler.local
 import java.nio.ByteBuffer
 
 import scala.concurrent.duration._
+import scala.language.postfixOps
```

Comment:
Yes that's the right thing, if you think that the use of postfix ops is justified.